pkg/interface/contacts/src/js/store.js

```diff
@@ -11,6 +12,7 @@ class Store {
     this.state = {
       contacts: {},
       groups: {},
+      channels: new Map,
```

Comment:
Is there a reason not to just call this associations?

Reply:
At first I had it as `metadata`, but realised that didn't explain itself to a dev trying to read this code later and went with "channels" because that's what this is ultimately for. I can see why the preferred tact could be just to keep the same terminology between front-end and back-end though, I can amend.